### PR TITLE
feat: Added temp brand tokens to mobile codebase

### PR DESCRIPTION
**Pull request by brianacnguyen**
## **Description**
- Added white and black brand color to be used for elements that need absolute white/black color regardless of theme

## **Related issues**

Fixes: N/A

## **Manual testing steps**

1. N/A

### **Before**
N/A

### **After**
N/A

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've clearly explained what problem this PR is solving and how it is solved.
- [x] I've linked related issues
- [ ] I've included manual testing steps
- [ ] I've included screenshots/recordings if applicable
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [x] I’ve properly set the pull request status:
  - [x] In case it's not yet "ready for review", I've set it to "draft".
  - [x] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: app/util/theme/temp-tokens/brandColors.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+/**
+ * Light theme colors
+ */
+
+export const brandColors: BrandColors = {
+  white: {
+    ['000']: '#FFFFFF',
+  },
+  black: {
+    ['000']: '#000000',
```
Comment (georgewrmarshall) [inline app/util/theme/temp-tokens/brandColors.ts:16]: I noticed that we're using square brackets for the property names here. Typically, square brackets are used for computed or dynamic property names. However, in this case, the property names '000' are static and I can't see a reason for them to be dynamic. Could you help me understand why we're using square brackets here? Is there a specific reason or requirement for this notation in this context?

**Review by georgewrmarshall — COMMENTED** [review]
Looking good! Those should align with the design tokens once we move them to the package. Left one comment regarding the square bracket key syntax

File: app/util/theme/temp-tokens/brandColors.ts (hunk 1 above)
Comment (brianacnguyen) [reply app/util/theme/temp-tokens/brandColors.ts:16]: you can't just use 000 as a prop name

Comment (brianacnguyen) [reply app/util/theme/temp-tokens/brandColors.ts:16]: my bad i mistook the import situation with not being able to import "000" objects. updated

**Review by georgewrmarshall — APPROVED** [review]
LGTM! Nice work 🚀

**Pull request merged**